Source/Project64-core/N64System/Mips/Mempak.cpp

```diff
@@ -63,16 +63,17 @@ void Mempak::Format(int32_t Control)
         0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0x01, 0xFF, 0x66, 0x25, 0x99, 0xCD,
         0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
         0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-        0x00, 0x71, 0x00, 0x03, 0x00, 0x03, 0x00, 0x03, 0x00, 0x03, 0x00, 0x03, 0x00, 0x03, 0x00, 0x03,
     };
 
-    memcpy(&Mempaks[Control][0], &Initialize[0], sizeof(Initialize));
+    memcpy(&Mempaks[Control][0], &page0[0], sizeof(page0));
 
-    for (size_t count = sizeof(Initialize); count < 128 * 256; count += 2)
+    for (size_t count = sizeof(page0) + 2; count < 128 * 256; count += 2)
```

Comment:
I could change from sizeof+2 to sizeof if you prefer (it will still work and get overwritten at the end), but that's all.